src/App.js

```diff
@@ -1,20 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.

Reply:
Thank you @gemmen29. Removed the Linter.